src/components/Editor.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, BoxProps } from '@chakra-ui/react';
+import { FunctionComponent } from 'react';
+import MonacoEditor, { monaco, MonacoEditorProps } from 'react-monaco-editor';
+
+import {
+  JSON_RPC_SCHEMA,
+  JSON_RPC_SCHEMA_URL,
+  SAMPLE_JSON_RPC_REQUEST,
+} from '../features/handlers/json-rpc/schema';
+
+export type EditorProps = MonacoEditorProps & BoxProps;
+
+/**
+ * Editor component. This uses Monaco Editor to provide a JSON editor.
+ *
+ * @param props - The props.
+ * @returns The editor component.
+ */
+export const Editor: FunctionComponent<EditorProps> = (props) => {
+  const handleMount = (editor: typeof monaco) => {
+    editor.languages.json.jsonDefaults.setDiagnosticsOptions({
+      validate: true,
+      schemas: [
+        {
+          uri: JSON_RPC_SCHEMA_URL,
+          fileMatch: ['*'],
+          schema: JSON_RPC_SCHEMA,
+        },
+      ],
+    });
+  };
+
+  return (
+    <Box
+      width="100%"
+      height="200px"
```

Comment:
Height should probably be a prop? Also, does it have to be static?

I feel like it could very quickly get too small for larger payloads currently.